docs/api/developer.md

```diff
@@ -246,5 +246,4 @@ Utility functions used by scvi-tools.
 
    utils.track
    utils.setup_anndata_dsp
-   utils.attrdict
```

Comment:
Removing the old one completely would be a breaking change. Can we just create a dummy attrdict object that inherits completely an ml collection?

Reply:
Addressed with new changes